pkg/email/email.go

```diff
@@ -15,6 +15,7 @@ import (
 type Config struct {
 	GRTEmail               models.EmailAddress
 	AccessibilityTeamEmail models.EmailAddress
+	RequesterEmail         models.EmailAddress
```

Comment:
I don't think we're using this anywhere, we don't need it. This config struct is for things that we set up when we initiate the server. The requester email changes email-to-email, so we'll want to pass that in on a per-email basis.